solutions/silver/cf-803D.mdx

```diff
@@ -0,0 +1,102 @@
+---
+id: cf-803D
+source: CF
+title: Magazine Ad
+author: Alex Du
+---
+
+<Spoiler title="Hint 1">
+
+Is there a way to format an ad of a given width using less than or equal to the maximum number of lines?
```

Comment:
hm given that this problem is bsearch
maybe you should point them towards that
"say you know that [this value] is/isn't possible. what can you deduce from this?"

Reply:
added hint 2

Reply:
i feel like that one should come before or replace this one tbh